pypolygames/evaluation.py

```diff
@@ -389,6 +389,10 @@ def run_evaluation(eval_params: EvalParams, execution_params: ExecutionParams, o
     print("#" + "EVALUATION".center(68) + "#")
     print("#" * 70)
 
+    # Print out the eval parameters for future reference
+    print("the eval parameters are")
+    print(eval_params)
```

Comment:
This kind of thing would be better served by using experiment-tracking software, like e.g. [Weights & Biases](https://wandb.ai), and saving the arguments in the run.

Reply:
Happy to merge for now though.

Reply:
I agree, and setting up wandb is in the plans. Still, there doesn't seem to be a downside to doing this for now.